Source/WebInspectorUI/UserInterface/Views/ButtonNavigationItem.js

```diff
@@ -221,16 +218,19 @@ WI.ButtonNavigationItem = class ButtonNavigationItem extends WI.NavigationItem
             switch (this._imageType) {
             case null:
             case WI.ButtonNavigationItem.ImageType.SVG: {
-                let glyphElement = WI.ImageUtilities.useSVGSymbol(this._image, "glyph");
-                glyphElement.style.width = this._imageWidth + "px";
-                glyphElement.style.height = this._imageHeight + "px";
-                this.element.appendChild(glyphElement);
+                if (this._image) {
+                    let glyphElement = WI.ImageUtilities.useSVGSymbol(this._image, "glyph");
+                    glyphElement.style.width = this._imageWidth + "px";
+                    glyphElement.style.height = this._imageHeight + "px";
+                    this.element.appendChild(glyphElement);
+                }
                 break;
             }
 
             case WI.ButtonNavigationItem.ImageType.IMG: {
                 let img = this.element.appendChild(document.createElement("img"));
```

Comment:
Should creating this element be inside the `if (this._image)`? For the SVG case we only create and append the element if we have `this._image`.

Reply:
This patch actually specifically wants a `<img>` even when `this._image` is not specified so that the `content` can be set in CSS.